integration_tests/__init__.py

```diff
@@ -82,6 +73,13 @@ def run_snapcraft(self, command, project_dir=None, yaml_dir=None,
         if not os.getenv('SNAPCRAFT_IGNORE_APT_AUTOREMOVE', False):
             self.addCleanup(self.run_apt_autoremove)
 
+        if os.path.exists('parts'):
+            self.parts_dir = 'parts'
+        if os.path.exists('stage'):
+            self.stage_dir = 'stage'
+        if os.path.exists('prime'):
+            self.prime_dir = 'prime'
```

Comment:
@sergiusens I know this is what you were after, but I don't like it very much.
I think we shouldn't put them in the base test case, because they will point to directories that don't yet exist.
And by putting them here in run_snapcraft, we are magically defining new attributes, which is not nice.

I'm more inclined to remove this commit. And even remove the project_dir argument, to force every test to call copy_project_to_cwd before calling run_snapcraft, and remove all the magical parts.